prov/verbs/src/verbs_info.c

```diff
@@ -1220,6 +1220,24 @@ vrb_info_add_dev_addr(struct fi_info **info, struct verbs_dev_info *dev)
 	return 0;
 }
 
+/* domain name may have a "-<xxx>" suffix */
+static inline int vrb_cmp_domain_and_dev_name(const char *domain_name,
+					      const char *dev_name)
+{
+	size_t cmp_len;
+	char *s = strchr(domain_name, '-');
```

Comment:
Is there any chance '-' could appear in the device name?  If so, strrchr() may work.

Reply:
Unless we could have 2 suffixes...

Reply:
Theoretically yes, but practically no.  I can change it to `strrchr()` just in case.

Reply:
The suffix is controlled by the provider. We only have one.